pkg/pb/orders.proto

```diff
@@ -60,6 +60,10 @@ message PieceHash {
     bytes hash = 2;
     // signature either satellite or storage node
     bytes signature = 3;
+    // size of uploaded piece
+    int64 piece_size = 4;
+    // timestamp when upload occur
+    google.protobuf.Timestamp timestamp = 5;
```

Comment:
Add `[(gogoproto.stdtime) = true]` like here: https://github.com/storj/storj/blob/15e74c8c3d68821eac13f4d2cccf89ba4604b237/pkg/macaroon/types.proto#L27

This will set the type to `time.Time` in the generated Go code, which is more convenient and we won't need to convert from Timestamp to Time.